test/e2e/performance/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubestellar performance regression testing
+This is a simple performance test that aims to identify major regressions. 
+
+The setup is a single bindingpolicy and a single deployment. The test adds the deployment to the WDS and measures the amount of time it takes to create the WDS object, the Binding object, the ManifestWork object, and the deployment object on the WEC. The starting time is just prior to adding the WDS object and the end time is just after the deployment object is observed on the WEC. The other timings are obtained from the objects creation timestamp. The test intentionally looks at seconds rather than milliseconds since we expect this test to run on VMs and and anything below seconds is likely just noise. In addition, the test uses creation timestamps which have seconds granuality.
```

Comment:
> ... adding the WDS object ...

I suspect you mean the _workload_ object here, the Deployment, rather than the _WDS_ object.

Reply:
Fixed